### PR TITLE
SOC2: Add asset management and 3rd party risk policies

### PR DESCRIPTION
**Pull request by knolleary**
Initial drafts of policies:

 - Asset Management
 - Third-Party Risk Management

### CODE REVIEW

File: src/handbook/company/security/asset-management.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+All personnel must immediately report the loss of any information systems,
+including portable or laptop computers, smartphones, PDAs, authentication tokens
+(keyfobs, one-time-password generators, or personally owned smartphones or
+devices with access to FlowForge systems) or other devices that can store and
+process or help grant access to FlowForge data.
+
+### Return of Assets
+
+All employees and third-party users of FlowForge equipment shall return all of
+the organizational assets within their possession upon termination of their
```
Comment (ZJvandeWeg) [inline src/handbook/company/security/asset-management.md:52]: Please link to this doc from the peopleops off boarding section.

File: src/handbook/company/security/asset-management.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+employment, contract, or agreement.
+
+### Handling of Assets
+
+Employees and users who are issued or handle FlowForge equipment are expected to
+use reasonable judgment and exercise due care in protecting and maintaining the
+equipment.
+
+Employees are responsible for ensuring that company equipment is secured and
```
Comment (ZJvandeWeg) [inline src/handbook/company/security/asset-management.md:61]: Sure, but let's document what the concrete steps are that must be taken.

File: src/handbook/company/security/asset-management.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+properly attended to whenever it is transported or stored.
+
+All mobile devices shall be handled in accordance with the [Information Security Policy](./information-security.md).
+
+## Exceptions
+
+Requests for an exception to this policy must be submitted to the CEO or CTO for
```
Comment (ZJvandeWeg) [inline src/handbook/company/security/asset-management.md:68]: All these policies are missing the how of this statement.

File: src/handbook/company/security/third-party-risk-management.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+To ensure protection of the organization's data and assets that are shared with,
+accessible to, or managed by suppliers, including external parties or third-party
+organizations such as service providers, vendors, and customers, and to maintain
+an agreed level of information security and service delivery in line with supplier
+agreements.
+
+## Scope 
+
+All data and information systems owned or used by FlowForge that are business
+critical and/or process, store, or transmit Confidential data.
```
Comment (ZJvandeWeg) [inline src/handbook/company/security/third-party-risk-management.md:22]: As it's capitalised, let's also link to the classification document. There's legal significance to the capital letter

File: src/handbook/company/security/third-party-risk-management.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+
+## Policy
+
+A list of approved vendors/partners must be maintained and reviewed annually. This
+list will be linked from the [Vendor section of the handbook](/handbook/operations/vendors/).
```
Comment (ZJvandeWeg) [inline src/handbook/company/security/third-party-risk-management.md:28]: Let's link in this PR

File: src/handbook/company/security/third-party-risk-management.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+own environment, meaning they cannot access, modify, or delete any data belonging
+to other third parties.
+
+Additional vendor agreements should be obtained as required by applicable regulatory
+compliance requirements.
+
+
+## Exceptions
+
+Requests for an exception to this policy must be submitted to the CEO or CTO for
```
Comment (ZJvandeWeg) [inline src/handbook/company/security/third-party-risk-management.md:48]: How?

File: src/handbook/company/security/third-party-risk-management.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+
+## Scope 
+
+All data and information systems owned or used by FlowForge that are business
+critical and/or process, store, or transmit Confidential data.
+
+
+## Policy
+
+A list of approved vendors/partners must be maintained and reviewed annually. This
```
Comment (ZJvandeWeg) [inline src/handbook/company/security/third-party-risk-management.md:27]: So we cannot merge this before we have such a list.

File: src/handbook/company/security/asset-management.md (hunk 1 above)
Comment (knolleary) [reply src/handbook/company/security/asset-management.md:52]: Done

File: src/handbook/company/security/asset-management.md (hunk 2 above)
Comment (knolleary) [reply src/handbook/company/security/asset-management.md:61]: I've added links to the Access Control policy regarding digital security and an explicit statement on not leaving equipment unattended in public spaces. I think that is sufficient in this context.

File: src/handbook/company/security/asset-management.md (hunk 3 above)
Comment (knolleary) [reply src/handbook/company/security/asset-management.md:70]: Agreed. Would a simple 'submitted via email' be sufficient to clarify? That will get a written record of the exception approval without having to create more process and gets this moving forward.

File: src/handbook/company/security/third-party-risk-management.md (hunk 5 above)
Comment (knolleary) [reply src/handbook/company/security/third-party-risk-management.md]: Done - spreadsheet created with an initial list based on the Saas Spend spreadsheet and added link to the vendor section.

File: src/handbook/company/security/third-party-risk-management.md (hunk 7 above)
Comment (knolleary) [reply src/handbook/company/security/third-party-risk-management.md:26]: We now have an initial list that lets us move forward.

**Comment by knolleary** [comment]
@ZJvandeWeg updated for your comments. Only outstanding question is the 'how' of the exception process. Have proposed  a simple change to clarify. If you're happy with that, I'll update all of the existing policy docs to match as part of this PR.

File: src/handbook/company/security/asset-management.md [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+properly attended to whenever it is transported or stored. Equipment must be
+secured in accordance with the [Access Control](./access-control.md) policy and
+must not be left unattended in public locations.
+
+All mobile devices shall be handled in accordance with the [Information Security Policy](./information-security.md).
+
+## Exceptions
+
+Requests for an exception to this policy must be submitted to the CEO or CTO for
+approval.
```
Comment (ZJvandeWeg) [inline src/handbook/company/security/asset-management.md:71]:
```suggestion
Requests for an exception to this policy must be submitted via email to the
CEO or CTO for approval.
```

File: src/handbook/company/security/third-party-risk-management.md [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+to other third parties.
+
+Additional vendor agreements should be obtained as required by applicable regulatory
+compliance requirements.
+
+
+## Exceptions
+
+Requests for an exception to this policy must be submitted to the CEO or CTO for
+approval.
```
Comment (ZJvandeWeg) [inline src/handbook/company/security/third-party-risk-management.md:48]:
```suggestion
## Exceptions

Requests for an exception to this policy must be submitted via email to the
CEO or CTO for approval.
```

**Review by ZJvandeWeg — APPROVED** [review]

**Review by ZJvandeWeg — APPROVED** [review]

**Pull request merged**